src/server/author.rs

```diff
@@ -0,0 +1,93 @@
+/*
+ * server/author.rs
+ *
+ * deepwell - Database management and migrations service
+ * Copyright (C) 2019 Ammon Smith
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+use chrono::prelude::*;
+use crate::prelude::*;
+use crate::author::{Author, AuthorType};
+use either::{Left, Either, Right};
+use diesel::Connection;
```

Comment:
I created `crate::service_prelude` so I didn't have to re-import the same things over and over again. Some of the stuff here that isn't in it, like `Either` could be added to it as well.

`crate::prelude` is more for external library consumers.